### DISCUSSION TITLE
Strange horizontal lines in Ptyxis terminal

### DISCUSSION THREAD

**Discussion by jeanas** [helix-editor/helix#12025] [Troubleshooting] [answered]
I just installed a fresh Fedora 41 system. I was previously using Fedora 40. In the [Helix](https://helix-editor.com/) editor (also installed from Fedora repositories), I'm seeing some very strange grey horizontal lines that were not present on Fedora 40 and which I cannot reproduce on GNOME Terminal if I install it on Fedora 41:

https://github.com/user-attachments/assets/d873d35e-dc0c-414b-95b8-ca264a1cfb7c

I'm not sure if this is a bug in Ptyxis or Helix. Any ideas?

[cross-post of [Ptyxis issue](https://gitlab.gnome.org/chergert/ptyxis/-/issues/264)]

**Comment by TornaxO7** [comment]
If I'm seeing that correctly, the issue doesn't seem to be on Helix side, right? ~~So can this be closed?~~ Nvm, this is a discussion xD
Eh, but somehow marking this question as answered would be still nice I think.

**Comment by 35359595** [comment]
I have same lines in Fedora 41 terminal in full screen (F11) and maximized (Sup + up) only when using gnome scaling (150%). With no scaling - not appearing. If terminal in scaled mode and in regular window (not maximized) - works fine.

**Comment by jlebon** [comment, marked as answer]
This is a bug that was fixed in the latest vte291 releases, but it's not in Fedora 41 yet. I filed https://bugzilla.redhat.com/show_bug.cgi?id=2331465 for that.

**Comment by jeanas** [comment]
Great, thank you!

**Reply by TornaxO7** [reply]
don't forget to mark the message as "answer", if it fixed your issue :D
